common/server.cpp

```diff
@@ -133,6 +133,8 @@ AuthenticationResult Server::loginUser(Server_ProtocolHandler *session, QString
                 delete se;
 
                 users.value(name)->prepareDestroy();
+            } else {
+                qDebug() << "Active session and sessions table inconsistent, please validate session table information for user " << name;
```

Comment:
So the issue is the fact that a previous (now dead) session exists in the db session table.  So we can not query for any old session data from memory.  In order to query for that stale information I would have to create a function to query the data in the db sessions table.  Would you like that done?  Right now we do not have that type of functionality and im not so sure its really needed since the username is stored in the db sessions table and will be in the log message.